packages/google_maps_flutter/google_maps_flutter_web/lib/src/overlay.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of google_maps_flutter_web;
+
+/// This wraps a [TileOverlay] in a [gmaps.MapType].
+class TileOverlayController {
+  /// For consistency with Android, this is not configurable.
+  // TODO(AsturaPhoenix): Verify consistency with iOS.
+  static const int logicalTileSize = 256;
```

Comment:
I am very curious about this `logicalTileSize`. From the API it seems that both a TileOverlay and each Tile can specify their size, so this implementation can do away with this `logicalTileSize` const.

Why is this fixed to 256, and do we need it?

(If this is 256, how come the example is [creating 100x100 tiles](https://github.com/flutter/packages/blob/main/packages/google_maps_flutter/google_maps_flutter/example/lib/tile_overlay.dart#L119-L120)? They look blurry on the web as they're scaled up to fit!)

(Also: can tiles be non-square??)

Reply:
@AsturaPhoenix yes, I used the physical size being `logicalSize * dpi` to generate high DPI tiles too. Let's go with consistency for now, having a fixed logical size.

(PS: In iOS they also mention 256x256 tiles or 512x512 for "retina" displays: https://developers.google.com/maps/documentation/ios-sdk/tiles#high_dpi_tiles_for_retina_devices, I'm not sure if the iOS SDK supports sizes any other than those two (like a 3x tile))